pylint_odoo/test/main.py

```diff
@@ -46,8 +46,8 @@
     'missing-newline-extrafiles': 4,
     'missing-readme': 1,
     'missing-return': 1,
-    'no-utf8-coding-comment': 3,
-    'unnecessary-utf8-coding-comment': 19,
+    'no-utf8-coding-comment': 6,
```

Comment:
Because old versions are ignoring paths without “__init__.py” files
But new versions includes all paths.
Then I have created a “__init__.py” file in order to be compatible with both versions and increasing the expected errors for this new folder